spec/services/order_line_item_creator_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+describe '.call' do
+  let(:order) { create(:order) }
+  let(:cart) { create(:cart, user: order.user) }
+  let(:item) { create(:cart_item, cart: cart) }
+  it 'create order line item with the order, item and shipping_costs attributes' do
```

Comment:
https://github.com/JRRGomes/refactoring-exercise/blob/159b4bf70c297600d5187fcacfaec038e7eff41c/app/services/order_line_item_creator.rb#L13-L19

Segundo a [api](https://api.rubyonrails.org/classes/ActiveRecord/Associations/CollectionProxy.html#method-i-create-21) o `create!` gera uma excessão quando o modelo é inválido. Como você está usando o `create!` nesse service, deve testar o caso da excessão também.

Segue a documentação do rspec para testes de excessões: https://relishapp.com/rspec/rspec-expectations/docs/built-in-matchers/raise-error-matcher